frontend/src/metabase/components/PopoverWithTrigger/ControlledPopoverWithTrigger.tsx

```diff
@@ -68,6 +69,8 @@ function ControlledPopoverWithTrigger({
     renderTrigger({ visible, onClick: handleTriggerClick })
   ) : (
     <TriggerButton
+      as="a"
+      role="button"
```

Comment:
If we don't include this the test https://github.com/metabase/metabase/runs/6331629671?check_suite_focus=true would fail.